api-client/src/endpoints/createFeedbackEndpoint.ts

```diff
@@ -19,56 +35,84 @@ export enum FeedbackType {
+
+export type FeedbackRouteType =
+  | CategoriesRouteType
+  | EventsRouteType
+  | PoisRouteType
+  | OffersRouteType
+  | DisclaimerRouteType
+  | SearchRouteType
+  | SprungbrettOfferRouteType
+
 export type ParamsType = {
-  feedbackType: FeedbackType
-  feedbackCategory?: FeedbackCategoryType
+  routeType: FeedbackRouteType
   city: string
   language: string
+  comment: string
+  contactMail: string
   query?: string
   slug?: string
+  searchTerm?: string
+  isPositiveRating: boolean | null
 }
-export type AdditionalParamsType =
-  | {
-      isPositiveRating: boolean
-      comment: string | null
-    }
-  | {
-      isPositiveRating: boolean | null
-      comment: string
-    }
-
-export default (baseUrl: string): Endpoint<ParamsType & AdditionalParamsType, void> =>
-  new EndpointBuilder<ParamsType & AdditionalParamsType, void>(FEEDBACK_ENDPOINT_NAME)
+
+const getFeedbackType = (routeType: FeedbackRouteType, slug?: string): FeedbackType => {
+  switch (routeType) {
+    case EVENTS_ROUTE:
+      return slug ? FeedbackType.event : FeedbackType.events
+
+    case OFFERS_ROUTE:
+      return slug ? FeedbackType.offer : FeedbackType.offers
+
+    case DISCLAIMER_ROUTE:
+      return FeedbackType.imprint
+
+    case POIS_ROUTE:
+      return slug ? FeedbackType.poi : FeedbackType.map
+
+    case CATEGORIES_ROUTE:
+      return slug ? FeedbackType.page : FeedbackType.categories
+
+    case SEARCH_ROUTE:
+      return FeedbackType.search
+
+    case SPRUNGBRETT_OFFER_ROUTE:
+      return FeedbackType.offer
+
+    default:
+      return FeedbackType.categories
+  }
+}
+
+export default (baseUrl: string): Endpoint<ParamsType, void> =>
+  new EndpointBuilder<ParamsType, void>(FEEDBACK_ENDPOINT_NAME)
     .withParamsToUrlMapper(params => {
-      const { city, language } = params
+      const { city, language, routeType, slug } = params
 
-      return `${baseUrl}/${city}/${language}/wp-json/extensions/v3/feedback/${params.feedbackType}/`
+      return `${baseUrl}/${city}/${language}/wp-json/extensions/v3/feedback/${getFeedbackType(routeType, slug)}/`
     })
-    .withParamsToBodyMapper((params: ParamsType & AdditionalParamsType): FormData => {
+    .withParamsToBodyMapper((params: ParamsType): FormData => {
+      const { isPositiveRating, comment, contactMail, query, searchTerm, slug } = params
       const formData = new FormData()
-      if (params.isPositiveRating !== null) {
-        formData.append('rating', params.isPositiveRating ? POSITIVE_RATING : NEGATIVE_RATING)
-      }
 
-      if (params.comment !== null) {
-        formData.append('comment', params.comment)
+      if (isPositiveRating !== null) {
+        formData.append('rating', isPositiveRating ? POSITIVE_RATING : NEGATIVE_RATING)
       }
 
-      if (params.query !== undefined) {
-        formData.append('query', params.query)
+      const queryWithSearchTerm = query === searchTerm ? query : `${searchTerm} (actual query: ${query})`
```

Comment:
Don't like how this looks in the backend. 
Maybe just the query in brackets without the "actual query"? Because the CMS is actually in german.

Reply:
This is also previous implementation 🙈